modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/ToolsEndpoint.java

```diff
@@ -366,7 +366,8 @@ public Response getVideoEditor(@PathParam("mediapackageid") final String mediaPa
     // Get workflows
     List<JValue> jWorkflows = new ArrayList<>();
     for (WorkflowDefinition workflow : getEditingWorkflows()) {
-      jWorkflows.add(obj(f("id", v(workflow.getId())), f("name", v(workflow.getTitle(), Jsons.BLANK))));
+      jWorkflows.add(obj(f("id", v(workflow.getId())), f("name", v(workflow.getTitle(), Jsons.BLANK)),
+              f("displayOrder", v(workflow.getDisplayOrder())));
```

Comment:
@rute-santos do this need a "Jsons.BLANK" as the third f param?

Reply:
No. The default display order is zero.

Reply:
My comment wasn't actually about the return value, I thought this had been throwing an error because of missing third param. Apparently not!, sorry.